aspnetcore/mvc/controllers/filters.md

```diff
@@ -403,6 +404,7 @@ The following sample exception filter uses a custom error view to display detail
 Exception filters:
 
 * Don't have before and after events.
+* Can't override the order of execution with the `Order` property.
```

Comment:
The `Order` value _does_ apply to `ExceptionFilter`s that implement `IOrderedFilter`. Unless we have a repro of a case where that isn't the case, this PR and #13239 should be closed. I'm dubious such a repro is possible.